vipps-ecom-api.md

```diff
@@ -235,6 +281,39 @@ that Vipps can call to delete the data. The Vipps app allows the user to later
 remove this consent (via the Profile -> Security -> "Access to your information"
 -> "Companies that remember you" screens).
 
+## Userinfo
+
+```
+Early draft, this should be considered pilot functionality that we are currently rolling out in our test environemnt
+```
+
+Vipps offers a functionality to ask for a generic conset to access Userinfo. This is based on the [Vipps Login](https://github.com/vippsas/vipps-login-api) solution, but you as a merchant can seemlessly combine the two functionalites in a single user session. Combining both the userinfo and payment elements. This means that a merchant needs both a Vipps Login integration, and a Vipps Ecom integration.
+
+When you initiate a payment add the parameter "Scopes:" to ask for a users consent to share these details. For example like be email, address and name. The scopes are based on [Login's list of socopes](https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-api.md#scopes).
+
+To request these scopes add the scopes to the initial call to
+[`POST:​/ecomm​/v2​/payments`](https://vippsas.github.io/vipps-ecom-api/#/Vipps%20eCom%20API/initiatePaymentV3UsingPOST)
+
+The user then consents and pays in the app.
+
+```
+This operation has considered an all or nothing approach, a user must complete a valid payment and consent to all
```

Comment:
why code block? if you want to emphasise use carrots `>` for blockquote

Reply:
Fixed